cirq-core/cirq/protocols/json_serialization.py

```diff
@@ -461,6 +490,8 @@ def has_serializable_by_keys(obj: Any) -> bool:
     """Returns true if obj contains one or more SerializableByKey objects."""
     if isinstance(obj, SerializableByKey):
         return True
+    if isinstance(obj, type):
```

Comment:
is there a reason why this can't just use the fall through?  I don't see how any of the other conditions would be met for obj being a type.